src/constants.js

```diff
@@ -0,0 +1,5 @@
+const LIKE_BAR_MIN_POWER = 125;
```

Comment:
We are using `config` package on `new-design` so it might be good idea to continue doing this on `dev` as well.